.github/workflows/deploy_check.yml

```diff
@@ -45,10 +45,14 @@ jobs:
       - name: Build and deploy to TestPyPI
         if: steps.duplicate_check.outputs.should_skip != 'true'
         run: |
-          CURRENT_VERSION=$(sed -n 's/__version__ = "\(.*\)"/\1/p' psydac/version.py)
+          CURRENT_VERSION=$(grep 'version *= *"\([0-9\.]*\)"' pyproject.toml | grep '[0-9\.]*' -oP)
```

Comment:
Do we need the `-P` flag? It does not seem to be supported on macOS

Reply:
```
  -o, --only-matching       show only nonempty parts of lines that match
  -P, --perl-regexp         PATTERNS are Perl regular expressions
```
It doesn't seem to be necessary. I get the same result without it